settings-release.xml

```diff
@@ -0,0 +1,22 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd">
+  <localRepository>/Users/gustavjulienne/.m2/repository</localRepository>
```

Comment:
What's this localRepository setting? It seems like it references a weird user?

Reply:
I have changed it to /home/jenkins/.m2 in the newest PR commit, this is outdated